config/environments/production.rb

```diff
@@ -77,3 +78,8 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 end
+
+Rails.application.config.middleware.use ExceptionNotification::Rack,
+  :slack => {
```

Comment:
So is this just setting up the possibility of connecting Slack with the Rails app or does this do just that already?